libsource/src/test/java/com/virtusize/libsource/TestFixtures.kt

```diff
@@ -31,10 +32,10 @@ internal object TestFixtures {
         """
```

Comment:
As I said we shouldn't parse those values

Reply:
I removed those test fixtures. See the commit 91c01dd